eth_keys/validation.py

```diff
@@ -15,21 +17,18 @@
 )
 
 
-def validate_integer(value):
-    # type: (int) -> None
+def validate_integer(value: Any) -> None:
```

Comment:
Do we really need to change the type to `Any` here and in the other validator functions below? It shouldn't be a problem as we raise an exception if the type is not the expected, but still it made me wonder...

Reply:
It seemed correct since this function is designed to allow *any* type, but validate that it's an integer.